metadata-proxy/nginx.conf

```diff
@@ -0,0 +1,47 @@
+user www-data;
+worker_processes 4;
+pid /run/nginx.pid;
+
+events {
+  worker_connections 20;
+}
+
+http {
+  server {
+    listen 127.0.0.1:2020;
```

Comment:
No, just curious. Can you use variables in the nginx configuration? It's not too bad since this is optional, but what if a customer has something running on that port? I would be nice to provide a way to override it.